---a/net/minecraft/stats/StatCrafting.java
+++b/net/minecraft/stats/StatCrafting.java
@@ -3,24 +3,19 @@
 import net.minecraft.item.Item;
 import net.minecraft.scoreboard.IScoreObjectiveCriteria;
 import net.minecraft.util.IChatComponent;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class StatCrafting extends StatBase
-{
+
+public class StatCrafting extends StatBase {
+
     private final Item field_150960_a;
     private static final String __OBFID = "CL_00001470";
 
-    public StatCrafting(String p_i45910_1_, String p_i45910_2_, IChatComponent p_i45910_3_, Item p_i45910_4_)
-    {
+    public StatCrafting(String p_i45910_1_, String p_i45910_2_, IChatComponent p_i45910_3_, Item p_i45910_4_) {
         super(p_i45910_1_ + p_i45910_2_, p_i45910_3_);
         this.field_150960_a = p_i45910_4_;
         int i = Item.getIdFromItem(p_i45910_4_);
 
-        if (i != 0)
-        {
+        if (i != 0) {
             IScoreObjectiveCriteria.INSTANCES.put(p_i45910_1_ + i, this.func_150952_k());
         }
     }
-
-    @SideOnly(Side.CLIENT)
-    public Item func_150959_a()
-    {
-        return this.field_150960_a;
-    }
-}
+}
